### PR TITLE
net-misc/r8152: New package

### PR DESCRIPTION
**Pull request by jayofdoom**
r8152 is the driver for many Realtek USB Ethernet NICs.

Signed-off-by: Jay Faulkner <jay@jvf.cc>

### CODE REVIEW

File: net-misc/r8152/r8152-2.16.3.ebuild [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+DESCRIPTION="r8152 driver for Realtek USB FE / GBE / 2.5G Gaming Ethernet Family Controller"
+HOMEPAGE="https://www.realtek.com/en/component/zoo/category/network-interface-controllers-10-100-1000m-gigabit-ethernet-usb-3-0-software"
+
+SRC_URI="http://rtitwww.realtek.com/rtdrivers/cn/nic1/${P}.tar.bz2"
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+RDEPEND="virtual/udev"
+DEPEND="$RDEPEND"
```
Comment (thesamesam) [inline net-misc/r8152/r8152-2.16.3.ebuild:17]: We use ${RDEPEND} style: https://projects.gentoo.org/qa/policy-guide/ebuild-format.html#pg0101

File: net-misc/r8152/metadata.xml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+		<name>Jay Faulkner</name>
+	</maintainer>
+	<maintainer type="project" proxied="proxy">
+		<email>proxy-maint@gentoo.org</email>
+		<name>Proxy Maintainers</name>
+	</maintainer>
+	<longdescription>Official Realtek r8152 linux driver. The following cards are currently supported:
+RTL8156 RTL8156B(S)(G) RTL8153 RTL8153B RTL8154 RTL8154B RTL8152B</longdescription>
+	<use>
+		<flag name="center-tap-short">Enable support for center tap short</flag>
```
Comment (thesamesam) [inline net-misc/r8152/metadata.xml:15]: What does this actually mean?

Comment (jayofdoom) [reply net-misc/r8152/metadata.xml:15]: This is verbatim from the upstream documentation. -> https://github.com/wget/realtek-r8152-linux/blob/master/ReadMe.txt#L31 I have no idea. I tried to figure it out, I still have no idea. The ebuild defaults to the behavior if you do no configuration at all.

File: net-misc/r8152/r8152-2.16.3.ebuild (hunk 1 above)
Comment (jayofdoom) [reply net-misc/r8152/r8152-2.16.3.ebuild:17]: ack; fixed

**Pull request closed without merging**